test_runner/fixtures/neon_fixtures.py

```diff
@@ -2301,6 +2301,20 @@ def extra_args(self) -> list[str]:
                 *["--uri", NeonProxy.link_auth_uri],
             ]
 
+    class LinkWithMetricsCollector(AuthBackend):
+        def __init__(self, metric_collection_endpoint: str, metric_collection_interval: str):
+            self.metric_collection_endpoint = metric_collection_endpoint
+            self.metric_collection_interval = metric_collection_interval
+
+        def extra_args(self) -> list[str]:
+            return [
+                # Link auth backend params
+                *["--auth-backend", "link"],
+                *["--uri", NeonProxy.link_auth_uri],
+                *["--metric-collection-endpoint", self.metric_collection_endpoint],
+                *["--metric-collection-interval", self.metric_collection_interval],
+            ]
```

Comment:
This is misguided. There's no such thing as `link auth with metrics collector`. `AuthBackend` should've been a union of Postgres/Link/whatever, but there's no such notion in python, so we have to use inheritance. Those two params are universal and should be provided in init (default is None).